src/sentry/users/api/endpoints/user_rollbacks.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers.base import serialize
+from sentry.models.rollbackuser import RollbackUser
+from sentry.users.api.bases.user import RegionSiloUserEndpoint
+from sentry.users.api.serializers.userrollback import UserRollbacksSerializer
+from sentry.users.services.user.model import RpcUser
+
+
+@region_silo_endpoint
+class UserRollbacksEndpoint(RegionSiloUserEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+    owner = ApiOwner.UNOWNED
```

Comment:
Marked this as unowned, but lmk if this should be something else. (maybe issues since most of the engineers working on this are from issues?)

Reply:
nevermind CI hates this, going to mark it as issues for instrumentation and alerting purposes then